### PR TITLE
Set cert-dir to prepare for 1.13

### PR DESCRIPTION
**Pull request by sttts**
Due to https://github.com/kubernetes/kubernetes/pull/69884.

### CODE REVIEW

**Comment by deads2k** [comment]
/lgtm

worked

**Comment by deads2k** [comment]
@sttts bootstrapping problem?

**Comment by soltysh** [comment]
/lgtm cancel
@deads2k this won't merge b/c `--cert-dir` is not known to k8s 1.12, we need your nuclear option (#80) to land rebase

**Comment by soltysh** [comment]
/hold
We'll need to land the rebase first before doing this.

**Comment by ravisantoshgudimetla** [comment]
@sttts - I am closing this one. Please re-open if you want this one.

**Comment by soltysh** [comment]
We will need it for after the rebase lands.
/reopen

**Comment by sttts** [comment]
Rebased landed.

/hold cancel

**Comment by sttts** [comment]
/cc @s-urbaniak 

Shouldn't this break metrics for scheduler?

File: bindata/v3.11.0/kube-scheduler/pod.yaml [hunk 1]
```diff
@@ -16,8 +16,7 @@ spec:
     command: ["hyperkube", "kube-scheduler"]
     args:
     - --config=/etc/kubernetes/static-pod-resources/configmaps/config/config.yaml
-    - --secure-port=0
-    - --port=10251
```
Comment (s-urbaniak) [inline bindata/v3.11.0/kube-scheduler/pod.yaml]: https://github.com/openshift/cluster-monitoring-operator/blob/172beb3e0d2cbf1f79bf61ba9c3f5bea373b4125/assets/prometheus-k8s/service-monitor-kube-scheduler.yaml#L15 relies on scraping the **insecure** port `10251` in the `openshift-kube-scheduler` namespace.

By ommitting `--port`, `hyperkube kube-scheduler` will still be listening on the insecure port `10251`. Just explicitely setting it to `--port=0` will disable the insecure port.

tldr: this _shouldn't_ break monitoring and I verified with a local scheduler that, having ommitted `--port` i stil get metrics on `127.0.0.1:10251/metrics`.

Comment (soltysh) [reply bindata/v3.11.0/kube-scheduler/pod.yaml]: @sttts I'm looking at our config which we point here and I don't see we set port to 10257 and insecure to 0, am I blind or something?
